src/Java/Lesson5/PassengerAuto.java

```diff
@@ -0,0 +1,43 @@
+package Java.Lesson5;
+
+import java.util.Scanner;
+
+public class PassengerAuto extends Terrestrial {
+
+    int body;
+    int numberOfPassengers;
+    static int timeInMotion;
+
+    public void traffic() {
+        Scanner in = new Scanner(System.in);
+        System.out.print("Введите время в пути (h): ");
+        timeInMotion = Integer.parseInt(in.nextLine());
+    }
```

Comment:
@Ihar-Malinouski  Перенедомудрил
```suggestion
    int timeInMotion;

    public int traffic() {
        Scanner in = new Scanner(System.in);
        System.out.print("Введите время в пути (h): ");
        return in.nextInt();
    }
```а потом уже присваивать....  timeInMotion = auto.trafic();